Packs/Rapid7_Nexpose/Integrations/Rapid7_Nexpose/Rapid7_Nexpose.py

```diff
@@ -1110,31 +1177,58 @@ def create_report(scope, name, template, report_format):
     }
 
     result = send_request(path, 'post', body=body)
-    return result['id'] if 'id' in result else None
+    return result.get('id')
 
 
-def download_report(report_id, name, report_format):
-    # Generate the report
-    path = 'reports/' + str(report_id) + ' /generate'
+def generate_report(report_id):
+    path = 'reports/{}/generate'.format(report_id)
     instance = send_request(path, 'post')
-    if (instance is None):
+    if not instance.get('id'):
```

Comment:
can `instance` be None here? if so, the `get` will raise an `AttributeError`

Reply:
No, if it's None then the request should fail.
Should I add a verification condition before?